src/corelibs/U2Lang/src/support/SimpleWorkflowTask.cpp

```diff
@@ -148,6 +149,14 @@ SimpleMSAWorkflow4GObjectTask::SimpleMSAWorkflow4GObjectTask(const QString &task
     MultipleSequenceAlignment al = MSAUtils::createCopyWithIndexedRowNames(msaObjectPointer->getMultipleAlignment());
 
     MultipleSequenceAlignmentObject *msaObject = MultipleSequenceAlignmentImporter::createAlignment(msaObjectPointer->getEntityRef().dbiRef, al, os);
+    for (const MultipleSequenceAlignmentRow &row : al->getMsaRows()) {
+        if (row->getData().contains("*")) {
+            msaObject->replaceAllCharacters('*', 'X');
+            ioLog.info(tr("Symbols '*' has been replaced with 'X' during alignment."));
+            break;
+        }
+    }
+    U2AlphabetUtils::assignAlphabet(al);
```

Comment:
I suppose, it's better to reassign alphabet only if you relapsed some nucleotides